graphene_django/views.py

```diff
@@ -299,51 +299,78 @@ def execute_graphql_request(
         except Exception as e:
             return ExecutionResult(errors=[e])
 
-        if request.method.lower() == "get":
-            operation_ast = get_operation_ast(document, operation_name)
-            if operation_ast and operation_ast.operation != OperationType.QUERY:
-                if show_graphiql:
-                    return None
+        operation_ast = get_operation_ast(document, operation_name)
```

Comment:
All of the old logic would at some point call `get_operation_ast` so I hoisted it up